WordPress/Classes/ViewRelated/NUX/LoginFields.h

```diff
@@ -0,0 +1,16 @@
+#import <Foundation/Foundation.h>
+
+@interface LoginFields : NSObject
```

Comment:
Since this class is soooo straightfoward, and it's purpose so easy to document, I'd suggest to give it a shot.  I think this particular class is an awesome example of an atomic / single-purpose class.